VERSION

```diff
@@ -1 +1 @@
-1.8.2-dev
+1.8.2+dev
```

Comment:
SemVer considers "1.8.2" and "1.8.2+whatever" to be the exact equivalent, so not sure if this is technically correct (if the intent is that "master" is more current than last release) https://semver.org/#spec-item-10

whereas "-dev" denotes a pre-release (so "lower than" the version number) (not great either because -dev > -beta (alphabetically)

Reply:
This is what we do in runc. The idea is to mark that this is version 1.8.2 plus some more commits (so SemVer interpretation is more or less correct -- 1.8.2+dev *is* 1.8.2).

I guess as long as this `+dev` suffix won't be used in the actual tag (which we do not plan to do), we're fine.